lib/service/service.go

```diff
@@ -1447,8 +1447,16 @@ func (process *TeleportProcess) initAuthService() error {
 			warnOnErr(tlsServer.Close(), log)
 		} else {
 			log.Info("Shutting down gracefully.")
```

Comment:
Should we change the message? "Shutting down ungracefully"? "Shutting down without grace"?  :P

```suggestion
			log.Info("Performing shutdown sequence (graceful shutdown not supported)")
```